components/Modal/ClaimBalanceBanner.tsx

```diff
@@ -0,0 +1,77 @@
+import { useState, useEffect, MouseEvent } from 'react';
+import { useAuthContext, useModalContext } from '../Provider';
+import {
+  Background,
+  ModalBox,
+  Section,
+  CloseIconContainer,
+  Title,
+  Description,
+  Row,
+  Spacer,
+  HalfButton,
+} from './Modal.styled';
+import ProtonSDK from '../../services/proton';
+import { formatPrice } from '../../utils';
+import { ReactComponent as CloseIcon } from '../../public/close.svg';
+
+export const ClaimBalanceBanner = (): JSX.Element => {
```

Comment:
Let's also rename this component to `ClaimBalanceModal` since the component is a modal.